### PR TITLE
feat: add transparent assets table

### PR DESCRIPTION
**Pull request by euharrison**
Add transparent assets table.

Add the masp banner.

Also, add a precision `2` to dollar values, to avoid huge numbers on the interface.

Screenshot 2024-11-06 at 19 39 43

### CODE REVIEW

File: apps/namadillo/src/App/Common/FiatCurrency.tsx [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import { Currency, CurrencyProps } from "@namada/components";
+import BigNumber from "bignumber.js";
 
 type FiatCurrencyProps = Omit<CurrencyProps, "currency">;
 
 export const FiatCurrency = (props: FiatCurrencyProps): JSX.Element => {
-  return <Currency currency={{ symbol: "$", fraction: "cents" }} {...props} />;
+  const formattedAmount = new BigNumber(props.amount.toPrecision(2));
```
Comment (emccorson) [inline apps/namadillo/src/App/Common/FiatCurrency.tsx:7]: This is also rounding bigger numbers e.g. `BigNumber(123)` displays as `$120`. Maybe the `decimalPlaces` function will work instead?

File: apps/namadillo/src/App/AccountOverview/TransparentOverviewPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+    return {
+      cells: [
+        <div key={`token-${display}`} className="flex items-center gap-4">
+          <div className="aspect-square w-8 h-8">
+            {icon ?
+              <img src={icon} />
+            : <div className="rounded-full h-full border border-white" />}
+          </div>
+          {display.toUpperCase()}
```
Comment (emccorson) [inline apps/namadillo/src/App/AccountOverview/TransparentOverviewPanel.tsx:44]: I think this should be `asset.symbol`.

File: apps/namadillo/src/App/AccountOverview/TransparentOverviewPanel.tsx [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+    <Panel className="min-h-[300px] flex flex-col" title="Transparent Overview">
+      {transparentTokensQuery.isPending ?
+        <SkeletonLoading height="100%" width="100%" />
+      : <AtomErrorBoundary
+          result={transparentTokensQuery}
+          niceError="Unable to load your transparent balance"
+          containerProps={{ className: "pb-16" }}
+        >
+          {transparentTokensQuery.data?.length ?
+            <>
```
Comment (emccorson) [inline apps/namadillo/src/App/AccountOverview/TransparentOverviewPanel.tsx:188]: The fragment could be removed here.

**Review by emccorson — APPROVED** [review]
Looks good to me! I just left a few minor comments.

**Comment by euharrison** [comment]
Closes #1172

**Pull request merged**